frontend/src/views/Landing/index.tsx

```diff
@@ -666,28 +667,29 @@ const LandingPage = (): JSX.Element => {
               </div>
             </div>
           </div>
-          {/* 4513(thuang): Comment out frameSrc for now until we figure out a compliant way to embed */}
-          {/* <div className={styles.freethinkVideoSection}>
+
+          <div className={styles.freethinkVideoSection}>
             <div className={styles.freethinkVideoSectionWrapper}>
               <div className={styles.freethinkVideoSectionTextContainer}>
                 <h2>Behind-the-scenes of Chan Zuckerberg CELLxGENE</h2>
                 <p>
-                  Watch a behind-the-scenes look at CZ CELLxGENE and explore how
-                  the platform can help scientists quickly surface important
-                  information that could lead to discoveries in treating
-                  disease.
+                  Watch a behind-the-scenes look at Chan Zuckerberg CELLxGENE
+                  (CZ CELLxGENE) and explore how the platform can help
+                  scientists quickly surface important information that could
+                  lead to discoveries in treating disease.
                 </p>
               </div>
-              <iframe
-                title="Wistia video player"
-                frameBorder="0"
-                scrolling="no"
-                className="wistia_embed"
-                name="wistia_embed"
-                src="https://fast.wistia.net/embed/iframe/fgzei2rjkl"
-              ></iframe>
+              <a
+                href="https://chanzuckerberg.com/blog/behind-the-scenes-of-chan-zuckerberg-cellxgene/"
+                target="_blank"
```

Comment:
for security, let's add `rel="noopener"` too!
```suggestion
                target="_blank"
```

Reply:
Updated! Thanks @tihuan